src/test/java/org/takes/facets/fallback/FbStatusTest.java

```diff
@@ -124,11 +126,11 @@ void sendsCorrectDefaultResponse() throws Exception {
             new FbStatus(code).route(req).get()
         );
         MatcherAssert.assertThat(
-            response.printBody(),
+            new BodyPrint(response).asString(),
```

Comment:
@baudoliver7 Please use `StartsWith` matcher instead of explicit call.

Reply:
@andreoss Good. But I suggest to use `TextIs` rather than `StartsWith` because we expect that response body content matches exactly.